doc/api.md

```diff
@@ -99,14 +99,15 @@ correspondence between the API resource fields and the same fields in the databa
 
 The following actions are available under the service group endpoint:
 
-##### POST /iot/services
+##### POST /iot/services or POST /iot/configGroups
 
 Creates a set of service groups for the given service and service path. The service and subservice information will
 taken from the headers, overwritting any preexisting values.
 
 Body params:
 
--   `services`: list of service groups to create. Each one adheres to the service group Model.
+-   `services`: list of service groups to create. Each one adheres to the service group Model (when `/iot/services` route is used).
+-   `configGroups`: list of service groups to create. Each one adheres to the service group Model (when `/iot/configGroups` route is used).
```

Comment:
I'll suggest to swap lines, as I think the right one is "configuration groups" and "service" is a kind of legacy that we want to deprecate. In addition, I'd suggest to add "(deprecated)" at the end of the line about services.

Reply:
Fixed in 424d59d